src/templates/teamMemberTemplate.js

```diff
@@ -9,102 +9,128 @@ import Blockquote from "./../components/atoms/Blockquote"
 import SectionTitle from "./../components/atoms/SectionTitle"
 import { existy, getFirstName } from "../helpers"
 
-export default function Template({ data, location }) {
-  const { markdownRemark } = data
-  const { frontmatter, html } = markdownRemark
-  const {
-    first_name,
-    name,
-    personal_pronouns,
-    role,
-    social,
-    medium_posts,
-    specialties,
-    image,
-    quote,
-  } = frontmatter
+class teamMemberTemplate extends React.Component {
+  componentDidMount() {
+    console.log("componentDidMount")
 
-  const memberLocation = frontmatter.location
-  const quoteName = getFirstName(first_name, name)
-  const personalPronouns = personal_pronouns ? personal_pronouns : ``
+    var playButton = document.getElementById("play-button")
+    playButton.onclick = function() {
+      console.log("click")
+      var player = document.getElementById("audio-player")
+      if (player.paused) {
+        // player.volume = .25;
+        player.play()
+      } else {
+        player.pause()
+      }
+    }
+  }
```

Comment:
I think this is fine and works great. It is not the most React-y way to do things but I don't think that matters so much here. If you wanted to refactor it at some point, I think the more react way to do it would be to make the audio player a component and mange the play/pause stuff in state.